### ISSUE TITLE
error report

### ISSUE THREAD

**Issue by gone** [avocode/avocode#1059]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.5
Source UID: 43838
```
TypeError: this._task.removeListener is not a function
  at t._unregisterTaskListeners (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/core/lib/downloader.js:1:3779)
  at t._restart (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/core/lib/downloader.js:1:2738)
  at t._send (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/core/lib/downloader.js:1:6454)
  at t._addRequest (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/core/lib/downloader.js:1:4400)
  at t.downloadFiles (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/core/lib/downloader.js:1:2016)
  at e.downloadFiles (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/core/lib/api.js:1:7802)
  at t.module.exports.t.downloadThumbnails (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/core/lib/assets-downloader.js:1:1032)
  at e._downloadThumbnails (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/core/lib/user.js:1:15507)
  at /home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/@avocode/core/lib/user.js:1:15090
  at /home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/async/lib/async.js:246:17
  at /home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/async/lib/async.js:122:13
  at _each (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/async/lib/async.js:46:13)
  at async.each (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/async/lib/async.js:121:9)
  at _asyncMap (/home/bbeecher/avocode/Avocode-linux-x64/resources/app.asar/node_modules/async/lib/
```

**Comment by connor-bean** [comment]
Hey, @gone -

Can you update to our most recent version of Avocode? You're using 2.4.5 and we have just released 2.9.1.

Linux 64 can be downloaded [here](https://avocode-media.s3.amazonaws.com/download/avocode-app/2.9.1/avocode-app-linux-2.9.1.zip)

Please let me know if this solves the issue.

Thanks,
Connor, Avocode Support

**Comment by jakubzitny** [comment]
As this is related to extremely old version, we can close this. Don't hesitate to let us know if you have any more issues or questions.
